runtime/compiler/control/JITServerHelpers.cpp

```diff
@@ -324,7 +333,7 @@ JITServerHelpers::packRemoteROMClassInfo(J9Class *clazz, J9VMThread *vmThread, T
    uintptr_t classChainOffsetOfIdentifyingLoaderForClazz = fe->sharedCache() ? 
       fe->sharedCache()->getClassChainOffsetOfIdentifyingLoaderForClazzInSharedCacheNoFail((TR_OpaqueClassBlock *)clazz) : 0;
 
-   return std::make_tuple(packROMClass(clazz->romClass, trMemory), methodsOfClass, baseClass, numDims, parentClass,
+   return std::make_tuple(packROMClass(clazz->romClass, trMemory, serializeClass), methodsOfClass, baseClass, numDims, parentClass,
```

Comment:
You don't need to pass `serializeClass` to `packROMClass`. Instead you could do:
`return std::make_tuple(serializeClass ? packROMClass(clazz->romClass, trMemory) : std::string(), methodsOfClass, baseClass, numDims, parentClass,`